hyrax.gemspec

```diff
@@ -64,6 +64,20 @@ SUMMARY
   spec.add_dependency 'mailboxer', '~> 0.12'
   spec.add_dependency 'nest', '~> 3.1'
   spec.add_dependency 'noid-rails', '~> 3.0.0'
+
+  # These should not be necessary, but without adding these as normal
```

Comment:
This may have been a local issue for me, I'm going to issue another commit removing these and see if these are indeed unnecessary.